package-lock.json

```diff
@@ -6,10 +6,8 @@
   "dependencies": {
```

Comment:
this file shouldnt be on the repository, we should delete it and add it to the .gitignore, other than that, the patch looks ok